src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -449,11 +459,23 @@ START_SECTION(PeakArea integratePeak(
   TEST_REAL_SIMILAR(pa.height, 966489.0)
   TEST_REAL_SIMILAR(pa.apex_pos, 2.7045)
   it = chromatogram.RTBegin(left);
-  for (Size i = 0; i < pa.hull_points.size(); ++i, ++it)
+  for (Size i = 0; i < pa.hull_points.size(); i += 4, it +=4)
   {
     TEST_REAL_SIMILAR(pa.hull_points[i][0], it->getRT())
     TEST_REAL_SIMILAR(pa.hull_points[i][1], it->getIntensity())
   }
+
+  STATUS("Integration type: simpson (1 point)")
+  pa = ptr->integratePeak(chromatogram, left, 2.478);
+  TEST_REAL_SIMILAR(pa.area, 881)
+  TEST_REAL_SIMILAR(pa.height, 881)
+  TEST_REAL_SIMILAR(pa.apex_pos, 2.477966667)
+
+  STATUS("Integration type: simpson (2 points)")
+  pa = ptr->integratePeak(chromatogram, left, 2.489);
+  TEST_REAL_SIMILAR(pa.area, 11.6081250000001)
+  TEST_REAL_SIMILAR(pa.height, 1384)
+  TEST_REAL_SIMILAR(pa.apex_pos, 2.488216667)
```

Comment:
I feel like that is a big problem if the peak area is substantially larger when integrating a single peak compared to two peaks - e.g. I integrate a small area and I get `881` and then I integrate a larger area and I get `11.6`. Maybe the correct behavior would be to throw an exception instead of producing two peak area estimates that are completely incomparable. 

The other approach would be to at least take the spacing of the next peak into account and multiply by that ? Basically just compute a rectangle in that case where the x-axis base is the distance to the next peak?